composer/utils/object_store/uc_object_store.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import annotations
```

Comment:
License header is missing. `pre-commit` adds it for you if you have missed it. So please run `pre-commit run -a` command before `git commit`. Thank you!